src/libs/pois.js

```diff
@@ -18,7 +17,7 @@ export function toAbsoluteUrl(poi) {
   const baseUrl = window.baseUrl;
   const lat = poi.latLon.lat.toFixed(7);
   const lon = poi.latLon.lng.toFixed(7);
-  const mapHash = `#map=${getBestZoom(poi.zoom)}/${lat}/${lon}`;
+  const mapHash = `#map=${getBestZoom(poi)}/${lat}/${lon}`;
```

Comment:
Next time, let's put that kind of bug fix in a separate PR, even if it's only one line.

Reply:
np!